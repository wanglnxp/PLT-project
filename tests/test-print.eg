int main()	{
	int a = 1;
	float b = 2.1;
	bool c = true;
	string d = " World!";
	print(0);
	print(a);
	print(0.23);
	print(b);
	print(false);
	print(c);
	print("Hello");
	print(d);

	return 0;
}
