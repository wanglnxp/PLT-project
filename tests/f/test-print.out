0
1
0.230000
2.100000
false
true
Hello
 World!
